src/Kmd.Logic.Gateway.Automation/PublishFile/Product.cs

```diff
@@ -1,11 +1,17 @@
 ﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
 
 namespace Kmd.Logic.Gateway.Automation.PublishFile
 {
     internal class Product
     {
+        private const string Value = "^[a-zA-Z0-9-]*$";
+
         public string Name { get; set; }
 
+        [RegularExpression(Value, ErrorMessage = "Key accepts only Alphanumeric characters and hypen")]
```

Comment:
Does anyone use it?

Reply:
This validation is removed from here and added in Lab